docs/source/guide/qse-1-intro.md

```diff
@@ -0,0 +1,260 @@
+        PauliString("XZZXI"),
+        PauliString("IXZZX"),
+        PauliString("XIXZZ"),
+        PauliString("ZXIXZ"),
+    ]
+
+    observable_in_code_space = Observable(FIVE_I)
+    all_paulis = projector_onto_code_space + [observable]
+    for g in all_paulis:
+        observable_in_code_space *= 0.5 * Observable(FIVE_I, g)
+    return observable_in_code_space
+
+def main():
+  circuit = prepare_logical_0_state_for_5_1_3_code()
+  executor = execute_with_depolarized_noise
+  observable = get_observable_in_code_space(PauliString("ZZZZZ"))
+```
+
+## 4. Check Operators and Code Hamiltonian
+We then assign the check operators as a list of pauli strings, and the code Hamiltonian as an Observable for the \[\[5,1,3\]\] code. The check operators of the \[\[5,1,3\]\] code are simply the expansion of the code’s 4 generators: $\[XZZXI, IXZZX, XIXZZ, ZXIXZ\]$
+
+```
+def get_5_1_3_code_check_operators_and_code_hamiltonian() -> tuple:
+    """
+    Returns the check operators and code Hamiltonian for the [[5,1,3]] code
+    The check operators are computed from the stabilizer generators:
+    (1+G1)(1+G2)(1+G3)(1+G4)  G = [XZZXI, IXZZX, XIXZZ, ZXIXZ]
+    source: https://en.wikipedia.org/wiki/Five-qubit_error_correcting_code
+    """
+    Ms = [
+        "YIYXX",
+        "ZIZYY",
+        "IXZZX",
+        "ZXIXZ",
+        "YYZIZ",
+        "XYIYX",
+        "YZIZY",
+        "ZZXIX",
+        "XZZXI",
+        "ZYYZI",
+        "IYXXY",
+        "IZYYZ",
+        "YXXYI",
+        "XXYIY",
+        "XIXZZ",
+        "IIIII",
+    ]
+    Ms_as_pauliStrings = [
+        PauliString(M, coeff=1, support=range(5)) for M in Ms
+    ]
+    negative_Ms_as_pauliStrings = [
+        PauliString(M, coeff=-1, support=range(5)) for M in Ms
+    ]
+    Hc = Observable(*negative_Ms_as_pauliStrings)
+    return Ms_as_pauliStrings, Hc
+
+
+def main():
+  circuit = prepare_logical_0_state_for_5_1_3_code()
+  executor = execute_with_depolarized_noise
+  observable = get_observable_in_code_space(PauliString("ZZZZZ"))
+  check_operators, code_hamiltonian = get_5_1_3_code_check_operators_and_code_hamiltonian()
+```
+
+
+## Run QSE
+Now we can run QSE. We first compute the noiseless result then the noisy result to compare to the mitigated result from QSE.
+
+
+```
+def main():
+    circuit = prepare_logical_0_state_for_5_1_3_code()
+  executor = execute_with_depolarized_noise
+  observable = get_observable_in_code_space(PauliString("ZZZZZ"))
+  check_operators, code_hamiltonian = get_5_1_3_code_check_operators_and_code_hamiltonian()
+  execute_with_qse(
+        circuit,
+        executor,
+        check_operators,
+        code_hamiltonian,
+        observable,
+    )
+```
+
+### The noiseless result
+```
+def execute_no_noise(circuit: QPROGRAM) -> np.ndarray:
+    return compute_density_matrix(
+        convert_to_mitiq(circuit)[0], noise_level=(0,)
+    )
+
+executor = execute_no_noise
+​​observable.expectation(circuit, execute_no_noise)
```

Comment:
```suggestion
​​observable.expectation(circuit, execute_no_noise)
```

Lets just pass the execute functions directly into  `observable.expectation` and `execute_with_qse` instead of defining the `executor` variable below.